src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -435,7 +442,7 @@ internal void SetOverrideRuntimeSupportsUnmanagedSignatureCallingConvention()
             => _overrideRuntimeSupportUnmanagedSignatureCallingConvention = true;
 
         internal bool RuntimeSupportsUnmanagedSignatureCallingConvention
-            => GetSpecialTypeMember(SpecialMember.System_Runtime_CompilerServices_RuntimeFeature__UnmanagedSignatureCallingConvention) is object
+            => RuntimeSupportsFeature(SpecialMember.System_Runtime_CompilerServices_RuntimeFeature__UnmanagedSignatureCallingConvention)
```

Comment:
This is probably going to conflict with https://github.com/dotnet/roslyn/pull/50845.

Reply:
(Not that I'm saying hold off, just that it's a race)